### ISSUE TITLE
are trained models available?

### ISSUE THREAD

**Issue by no-zzz-un** [commaai/research#35]
has anyone made the trained models available online?

**Comment by tnedev** [comment]
The keras model output is here. I guess the second file is the wights which you need: https://github.com/commaai/research/tree/master/outputs/steering_model

**Comment by no-zzz-un** [comment]
@tnedev that's only the steering model? i want a trained generative model..

**Comment by no-zzz-un** [comment]
it would even be great to hear from someone, how long did training the generative model take you with whatever hardware configuration..

**Comment by zhaohuaqing1993** [comment]
DId you train it successfully?

**Comment by no-zzz-un** [comment]
@zhaohuaqing1993 no, i haven't actually tried. i was hoping to get someone else's report of time+hardware first.
